apps/vs-code-designer/src/app/commands/createNewCodeProject/createNewCodeProject.ts

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { funcVersionSetting, projectLanguageSetting, projectOpenBehaviorSetting, projectTemplateKeySetting } from '../../../constants';
+import { localize } from '../../../localize';
+import { addLocalFuncTelemetry, tryGetLocalFuncVersion, tryParseFuncVersion } from '../../utils/funcCoreTools/funcVersion';
+import { getGlobalSetting, getWorkspaceSetting } from '../../utils/vsCodeConfig/settings';
+import { OpenBehaviorStep } from '../createNewProject/OpenBehaviorStep';
+import { FolderListStep } from '../createNewProject/createProjectSteps/FolderListStep';
+import { OpenFolderStepCodeProject } from './CodeProjectBase/OpenFolderStepCodeProject';
+import { NewCodeProjectTypeStep } from './createCodeProjectSteps/NewCodeProjectTypeStep';
+import { setWorkspaceName } from './createCodeProjectSteps/SetWorkspaceName';
+import { setMethodName } from './createCodeProjectSteps/createFunction/setMethodName';
+import { setNameSpace } from './createCodeProjectSteps/createFunction/setNamepSpace';
```

Comment:
Looks like a typo here in `setNamepSpace`

```suggestion
import { setNameSpace } from './createCodeProjectSteps/createFunction/setNamesSpace';
```